source/installation/install-software.rst

```diff
@@ -91,55 +92,51 @@ software requirements:
 
             sudo apt install ondemand
 
-.. note::
-
-   If *authenticating against LDAP* or *wishing to evaluate OnDemand* using `ood` user, you **must install** `ondemand-dex` as 
-   described below. See :ref:`add-ldap` for details on configuration of LDAP and :ref:`start-services` for the `ood` user.
-
-#. (Optional) Install :ref:`authentication-dex` package
+3. Start services
+-----------------
 
-   **RedHat/CentOS/Rocky only**
+   .. tabs::
 
-    .. code-block:: sh
+      .. tab:: RHEL/CentOS 7
 
-       sudo yum install ondemand-dex
+        .. code-block:: sh
 
-   **Ubuntu only**
+          sudo systemctl start httpd24-httpd
+          sudo systemctl enable httpd24-httpd
 
-    .. code-block:: sh
 
-       sudo apt install ondemand-dex
+      .. tab:: RHEL/CentOS/Rocky 8
 
-#. (Optional) Install OnDemand SELinux support if you have SELinux enabled. For details see :ref:`ood_selinux`
```

Comment:
Do we maybe want some mention of SELinux at bottom of install instruction similar to thing about authentication? A lot of sites end up using SELinux and I want to make sure it's called out somewhere in the installation docs, even if that's just a link to the SELinux page.

Reply:
Yea, good catch this should be called out somewhere.

Reply:
Added in a few places. plus I got rid of the TOC header for space.